src/config/snap.js

```diff
@@ -1,3 +1,4 @@
+// REVIEW: This file appears to be currently unused. Candidate for removal

```

Comment:
It's not currently unused. 'snap.js' uses it. It's just in the wrong place. The data needs to be re-named to be more descriptive, and objects need to be moved to the data folders with some sensible file naming. Then snap.js needs to be updated to use the new locations and names.

Reply:
This is my mistake - you're correct. It certainly is used. Not sure why I left this comment. I did update the import as part of this PR, as seen farther down the page.

I will remove this line comment.